doc/todo/RELEASE

```diff
@@ -25,238 +13,66 @@ Following shell expansions are used in this document:
-cd oyranos
-mkdir build
-cd build
-cmake .. # might fail, simply comment out missing libs
-make -j5 # not needed
-make test2 # or make test-2 for the most of the tests
-src/tests/test2 Elektra
-
-
-== Documentation ==
-
-And the API docu:
- mkdir ~elektra-doc/api/$VERSION/
- cd ~build && cmake -DBUILD_PDF=ON . && rm -rf doc && make html man && cp -ra doc/html doc/latex doc/man ~elektra-doc/api/$VERSION/
-
-Symlink current to latest version and add everything:
- cd ~elektra-doc/api/ && rm current && ln -s $VERSION current && git add current $VERSION && git commit -a -m "$VERSION Release"
-
-
-
+- https://build.libelektra.org/job/libelektra/job/master/ must pass
+- Trigger release pipeline with dry run
+- Inspect the logs from the pipeline
+  - [ ] Error logs (*.error) must be empty
+  - [ ] Check in the (test-*) folders if all tests passed
+  - [ ] Check if version is correct in the version file
+  - [ ] Check if the debian package build log was successfull (debian/<DEBIAN_CODENAME>/elektra_*.buildinfo)
+- Check if all artifacts are present
+  - [ ] Check debian/<DEBIAN_CODENAME> if all debian packages were built
+  - [ ] Unarchive the elektra tarball and briefly verify if the correct version was archived
+- Inspect the changes to the libelektra git repository (*.bundle)
+  Inspect the git log of the master and debian branch with: git log -p --author="jenkins-release"
+  Master:
+  - [ ] Check diff of commit "Regenerate dot of plugins for release <VERSION>" and if 
+        doc/images/plugins.pdf was generated correctly (Only if plugins were modified)
+  - [ ] Check diff of commit "Update contract for release <VERSION>": Plugin tags in the plugin 
+        README.md should be sorted (Only if tags are not sorted)
+  Debian:
+  - [ ] Check if master is merged to debian branch (commit: "merge <VERSION>")
+  - [ ] Check diff of commit "Debian Package <VERSION>-1 (UNRELEASED)":
+        An entry for a new upstream version should have been created in debian/changelog
+  - [ ] Check diff of commit "Debian Package <VERSION>-1":
+        The previously created version entry in debian/changelog should have been updated from
+        UNRELEASED to unstable. 
+- Inspect the changes to the docu git repository (docu.bundle)
+  - [ ] Check diff of the latest commit "<VERSION> release" with: git log -p 
+- Inspect the website from the pipeline
+
+== Preperation ==
+
+- Finalize release notes without hash sums (move to final name, add links where to download release)
+- cp doc/todo/NEWS.md doc/news/_preparation_next_release.md
 
 = Publish =
 
-== Debian Package ==
-
- cd ~elektra/$VERSION/debian/$DVERSION && reprepro --ignore=wrongdistribution include stretch elektra_${DVERSION}_`dpkg-architecture -qDEB_BUILD_ARCH`.changes && update-reprepro
-
-== Commit source package ==
+- Trigger release pipeline without dry run
+- Add hash sums to release notes and run linkchecker to verify that all download links are working:
```

Comment:
Such checks should have happened earlier!

Reply:
Linkchecker is already run inside the pipeline. Adding the hash sums can only be done after the package is built and the linkchecker is only run to verify if the download links are correct after publishing. Maybe I understand something wrong, but I don't think running these checks is possible before publishing.

Reply:
I also published the tarballs, docs, etc and then I moved the release notes to the final place with the version number due to this exact reason.

Reply:
Sorry, I misunderstood the sentence. I didn't realize that the "download links" refer to the links in the release notes. Yes, so everything is all right as is.